src/test/java/org/wise/portal/service/peergroup/impl/PeerGroupMembershipServiceImplTest.java

```diff
@@ -56,6 +56,15 @@ public void addMember_MemberOfAnotherPeerGroup_RemoveMemberFirst() {
     verify(peerGroupDao);
   }
 
+  @Test
+  public void removeMember_ReturnUpdatedGroup() {
+    expectSavePeerGroup(1);
+    replay(peerGroupDao);
+    PeerGroup peerGroup = service.removeMember(testHelper.peerGroup1, run1Workgroup1);
+    assertEquals(2, peerGroup.getMembers().size());
+    verify(peerGroupDao);
+  }
+
```

Comment:
The test passes right now but shouldn't the test expect the peerGroup to only have 1 member after removing one?

Also, should it assert that the number of members be a certain value before and after the function call like this?
```
assertEquals(2, peerGroup.getMembers().size());
PeerGroup peerGroup = service.removeMember(testHelper.peerGroup1, run1Workgroup1);
assertEquals(1, peerGroup.getMembers().size());
```